src/ember-app.js

```diff
@@ -116,21 +118,34 @@ class EmberApp {
   buildWhitelistedRequire(whitelist, distPath) {
     whitelist.forEach(function(whitelistedModule) {
       debug("module whitelisted; module=%s", whitelistedModule);
+
+      // Remove after 2.0.0
+      let packageName = getPackageName(whitelistedModule);
+
+      if (packageName !== whitelistedModule) {
+        console.warn("[DEPRECATION] Whitelisting module path '" + whitelistedModule + "' will not be supported after fastboot#2.0.0 and it should be removed from the whitelist.");
```

Comment:
I think in fastboot we have version maintainence between `ember-cli-fastboot` and `fastboot`. You can see  here: https://github.com/ember-fastboot/fastboot/blob/master/src/fastboot-schema-versions.js.  I think we should use that versus version of fastboot. This can be confusing as `ember-cli-fastboot` is already on 2.0.0.

Reply:
Thanks 👍, will update accordingly.